Userland/Libraries/LibWeb/Layout/Box.h

```diff
@@ -65,6 +69,7 @@ class Box : public NodeWithStyleAndBoxModelMetrics {
     Optional<CSSPixels> m_natural_height;
     Optional<CSSPixelFraction> m_natural_aspect_ratio;
 
+    Vector<JS::NonnullGCPtr<Node>> m_contained_children;
     Vector<JS::NonnullGCPtr<Node>> m_contained_abspos_children;
```

Comment:
allocating a vector of contained children for each box seems quite wasteful. adding IntrusiveListNode in a Layout::Node that forms linked list of contained nodes might be a better idea.

Reply:
done 👍